api/src/main/java/org/apache/cloudstack/api/response/TrafficTypeResponse.java

```diff
@@ -56,6 +58,17 @@ public class TrafficTypeResponse extends BaseResponse {
     @Param(description = "The network name label of the physical device dedicated to this traffic on a HyperV host")
     private String hypervNetworkLabel;
 
+    // why not VLAN_ID
```

Comment:
clarify

Reply:
@madhukar93 sorry didn't get your comment; if it's by use of the term - vlan and vlan-id are interchangeable.

Reply:
left it as a note to myself, but thank you for the clarification.